app/models/paymentAllocationCharges/PaymentAllocationViewModel.scala

```
@@ -37,7 +41,7 @@ case class PaymentAllocationViewModel(paymentAllocationChargeModel: FinancialDet
     paymentAllocationChargeModel.documentDetails.exists(_.credit.isDefined)
 
   def getEffectiveDateOfPayment: LocalDate =
-      paymentAllocationChargeModel.documentDetails.head.effectiveDateOfPayment.getOrElse(throw MissingFieldException("Effective Date Of Payment"))
+    paymentAllocationChargeModel.documentDetails.head.effectiveDateOfPayment.getOrElse(throw MissingFieldException("Effective Date Of Payment"))
```

if we can throw an error when there is no documentsDetails present?

---

Done !